snap-build/amd64/run_qemu.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I think it might be simpler to use the osbuilder approach and create a single `snap-build/run_qemu.sh` script and then have a `snap-build/${arch}/config.sh` file that defines some variables for the particular architecture that `snap-build/run_qemu.sh` will use. That will minimise duplication and make it easier to add support for additional architectures.